templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
+COMMITS=$(($COMMITS))
```

Comment:
I think this is because we have to force this to be a number for it to work correctly?